src/solver/variable/state.cpp

```diff
@@ -73,14 +76,14 @@ State::State(Data::Study& s) :
 {
 }
 
-void State::initFromThermalClusterIndex(const uint clusterAreaWideIndex, uint numSpace)
+void State::initFromThermalClusterIndex(const uint clusterAreaWideIndex)
 {
```

Comment:
**numberOfUnitsONbyCluster** : same thing. We should make it a current function local **double** variable, and initialize to zero before using it.